packages/labs/examples/selectExample.tsx

```diff
@@ -53,6 +57,7 @@ export class SelectExample extends BaseExample<ISelectExampleState> {
                 <Button
                     rightIconName="caret-down"
                     text={film ? film.title : "(No selection)"}
+                    disabled={disabled}
```

Comment:
The need for a `disabled` prop on both the `Select` and the child could be a sticking point for consumers.  @llorca - can you come up with messaging to communicate the need for both? I'm thinking it could be a `pt-primary` info callout or something, probably just below the **Javascript API** heading. Initial idea for language:

> ### Disabling a Select
> Because a `Select` can accept arbitrary content for the trigger, disabling the component requires passing `disabled={true}` (or equivalent) to both the `Select` and its child, separately.

Reply:
Nice, let's make it a primary callout. I'd put it in between the initial docs and example, since that's where the target is mentioned in the first place:
![image](https://user-images.githubusercontent.com/199754/30286414-61a773be-96d6-11e7-8829-f29dcc624db0.png)

Small edit:
> and its target child